docs/4.x/tagmanager/parameters.md

```diff
@@ -120,6 +120,28 @@ $field->transform = function ($value, Setting $setting) {
 }
 ```
 
+### Migrating tags to include new parameters
+
+When you add a new parameter to a tag template, it can cause some strange behavior if the existing tags of the template type aren't migrated. To aid in this, there is a helper class that can be used in migration scripts.
```

Comment:
@snake14 Strange behaviour won't highlight what is the issue, its better to mention what issue it will create as the user won't have context like us
```suggestion
When you add a new parameter to a tag template, all the existing tags referencing this template will be shown as changed. To avoid this, there is a helper class that can be used in migration scripts.

Reply:
@AltamashShaikh Thank you. That's good feedback. I've tried to incorporate your suggestion.